Lombiq.Tests.UI/Models/ElasticsearchRunningContext.cs

```diff
@@ -17,93 +16,109 @@ namespace Lombiq.Tests.UI.Models;
 public record ElasticsearchRunningContext(Guid Id, string Prefix)
 {
     /// <summary>
-    /// Gets the expression that refers to all indexes that starts with <see cref="Prefix"/>. This should only be used
+    /// Gets the expression that refers to all indexes that start with <see cref="Prefix"/>. This should only be used
     /// with <see cref="IElasticClient"/>, because the OrchardCore-specific services automatically apply the prefix from
     /// configuration so it would result in double prefixing.
     /// </summary>
     private IndexName LowLevelIndexName => Indices.Index($"{Prefix}_*");
 
+    // Elasticsearch indexing sometimes takes longer, and the testing starts before it finishes. To prevent that, we are
+    // checking if all of the indexing tasks are finished.
```

Comment:
The phrasing of "the testing starts before it finishes" is unclear. The "it" reads like we are still talking about the testing, as in "the testing starts before the testing finishes".
```suggestion
    // Elasticsearch indexing sometimes takes longer, and the testing starts before indexing finishes. To prevent that,
    // we are checking if all indexing tasks are finished.
```